app/src/main/java/com/fstyle/androidtrainning/data/local/dao/MoviesDAO.java

```diff
@@ -0,0 +1,32 @@
+package com.fstyle.androidtrainning.data.local.dao;
+
+import android.arch.persistence.room.Dao;
+import android.arch.persistence.room.Delete;
+import android.arch.persistence.room.Insert;
+import android.arch.persistence.room.OnConflictStrategy;
+import android.arch.persistence.room.Query;
+import com.fstyle.androidtrainning.data.local.entity.MovieEntity;
+import com.fstyle.androidtrainning.data.model.Movie;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.data.model.Movie. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports